src/apify_client/clients/resource_clients/task.py

```diff
@@ -0,0 +1,214 @@
+        )
+
+        response = self.http_client.call(
+            url=self._url('runs'),
+            method='POST',
+            headers={'content-type': 'application/json; charset=utf-8'},
+            json=task_input,
+            params=request_params,
+        )
+
+        return _parse_date_fields(_pluck_data(response.json()))
+
+    def call(
+        self,
+        *,
+        task_input: Optional[Dict[str, Any]] = None,
+        build: Optional[str] = None,
+        memory_mb: Optional[int] = None,
+        timeout_secs: Optional[int] = None,
+        wait_for_finish: Optional[int] = None,
+        webhooks: Optional[List[Dict]] = None,
+    ) -> Dict:
+        """Start a task and wait for it to finish before returning the Run object.
+
+        It waits indefinitely, unless the wait_secs argument is provided.
+
+        https://docs.apify.com/api/v2#/reference/actor-tasks/run-collection/run-task
+
+        Args:
+            task_input (dict, optional): Task input dictionary
+            build (str, optional): Specifies the actor build to run. It can be either a build tag or build number.
+                                   By default, the run uses the build specified in the task settings (typically latest).
+            memory_mb (int, optional): Memory limit for the run, in megabytes. By default, the run uses a memory limit specified in the task settings.
+            timeout_secs: (int, optional): Optional timeout for the run, in seconds. By default, the run uses timeout specified in the task settings.
+            wait_for_finish: (bool, optional): The maximum number of seconds the server waits for the run to finish.
+                                               By default, it is 0, the maximum value is 300.
+            webhooks (list, optional): Specifies optional webhooks associated with the actor run, which can be used to receive a notification
+                                       e.g. when the actor finished or failed. Note: if you already have a webhook set up for the actor or task,
+                                       you do not have to add it again here.
+
+        Returns:
+            The run object
+        """
+        raise ApifyClientError('Method not yet finished. Run subclient needs to be implemented first.')
+
+        # run = self.start(
+        #     task_input=task_input,
+        #     build=build,
+        #     memory_mb=memory_mb,
+        #     timeout_secs=timeout_secs,
+        #     wait_for_finish=wait_for_finish,
+        #     webhooks=webhooks,
+        # )
+
+        # TODO - retrieve the run using Run client and wait on it
+
+    def get_input(self) -> Optional[Dict]:
+        """Retrieve the input for this task.
+
+        https://docs.apify.com/api/v2#/reference/actor-tasks/task-input-object/get-task-input
+
+        Returns:
+            Retrieved task input
+        """
+        try:
+            response = self.http_client.call(
```

Comment:
Maybe there could be a BaseClient or ResourceClient function for this call? (if this kind occurs in other endpoints too)

Reply:
Same for the update_input function, of course.

Reply:
Let's leave this to a separate PR if we decided to do that